gulpfile.babel.js

```diff
@@ -243,56 +186,29 @@ gulp.task('scripts', ['lint'], () => {
       sourceMapIncludeSources: true
     }))
     .pipe($.header(banner, {pkg}))
-    .pipe($.concat('material.min.js'))
+    .pipe($.rename({extname: '.min.js'}))
     // Write Source Maps
     .pipe($.sourcemaps.write('.'))
-    .pipe(gulp.dest('dist'))
-    .pipe($.size({title: 'scripts'}));
-});
-
-// Clean Output Directory
-gulp.task('clean', () => del(['dist', '.publish']));
```

Comment:
`.publish` is not a thing anymore so it’s effectively deleting `dist`. I can re-add if desired.